src/twisted/internet/test/test_endpoints.py

```diff
@@ -4196,7 +4221,7 @@ def test_hostnameEndpointConstruction(self):
         self.assertEqual(hostnameEndpoint._hostBytes, b"example.com")
         self.assertEqual(hostnameEndpoint._port, 443)
         self.assertEqual(hostnameEndpoint._timeout, 10)
-        self.assertEqual(hostnameEndpoint._bindAddress, nativeString("127.0.0.1"))
+        self.assertEqual(hostnameEndpoint._bindAddress, (nativeString("127.0.0.1"), 0))
```

Comment:
Do we need `nativeString` here ?




```suggestion
        self.assertEqual(hostnameEndpoint._bindAddress, "127.0.0.1", 0))
```

I have checked the implementation of nativeString and I don't see why we need it here.

https://github.com/twisted/twisted/blob/075191d3fd03193e48247fff39e108c784962da2/src/twisted/python/compat.py#L329-L344

Reply:
I think we don't, I'll remove it and the other nearby uses.